src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.ballColor = color;
+        this.ballNumber = number;
+    }
+
+    public String getBallColor() {
+        return this.ballColor;
+    }
```

Comment:
```suggestion
    public String getColor() {
        return color;
    }
```
this. is redundant here